azuredevops/internal/service/taskagent/resource_kubernetes_resource.go

```diff
@@ -0,0 +1,205 @@
+package taskagent
```

Comment:
rename `resource_kubernetes_resource.go` to `resource_environment_resource_kubernetes.go`